crypto/fipsmodule/CMakeLists.txt

```diff
@@ -163,10 +163,17 @@ if (S2N_BIGNUM_DIR AND UNIX AND NOT APPLE)
     bignum_tomont_p384.S
     bignum_deamont_p384.S
     bignum_montmul_p384.S
+    bignum_montmul_p384_alt.S
     bignum_montsqr_p384.S
+    bignum_montsqr_p384_alt.S
```

Comment:
Does this mean these _alt functions are included in the arm build? If yes, are they needed now?

Reply:
yes they are included but not used right now, I explained in the comment here: https://github.com/awslabs/aws-lc/pull/386/files#diff-69efd8604b81a4adc44b8fa275c47b60ed2c615cc38fe1b7e345b08871a3b711R82-R87

Two reasons I included them: simplifies the code because if I exclude those we'd have to have ifdefs to differentiate x86 and arm, and second, I was hoping we soon find a way to make use of them (i.e., detect the platforms the alternatives are faster on).

Would you prefer I remove them?

Reply:
Thanks for the explanation. May be then we can put a comment after l.172: 
"The following files contain functions of the same name for x86_64. However, on AArch64, these functions are defined as the same "non-alt" functions in their respective files, see third_party/s2n-bignum/arm/p384/bignum_<deamont|tomont>_p384.S"

Reply:
Done, thanks!